build.sh

```diff
@@ -128,16 +128,14 @@ passwd --lock root
 groupadd -r autologin
 useradd -m ${USERNAME} -G autologin,wheel
 echo "${USERNAME}:${USERNAME}" | chpasswd
-echo "
-root ALL=(ALL) ALL
-${USERNAME} ALL=(ALL) ALL
```

Comment:
does removing this mean the user cannot use `sudo`? Or is there a default file?

Reply:
By default it should already include a sudoers file. I haven't built an image to verify...

Reply:
Okay, well, I guess we'll find out.